Monolog/Processor/LowerCaseLevelNameProcessor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace DigipolisGent\SyslogBundle\Monolog\Processor;
+
+/**
+ * Processor that adds the lowercase level name to a log record.
+ */
+class LowerCaseLevelNameProcessor
+{
+
+    /**
+     * Adds the lowercase level name to the record.
```

Comment:
Documentation: it doesn't actually _add_ the level name to the record, it transforms it.